### PR TITLE
exporter: add postgres_exporter

### LINKED ISSUE #36: Add postgres_exporter for Prometheus
If PostgreSQL server is used, this purpose is to install [postgres_exporter](https://github.com/prometheus-community/postgres_exporter) to monitor DB with Prometheus.

### PR DESCRIPTION
**Pull request by kpgtek**
#### Which issue(s) this PR fixes

Fixes #36

#### Agreements

- [x] I hereby declare this contribution to be licensed under the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0).
- [x] I hereby agree to grant [TOSIT](https://www.tosit.io/) a copyright license to use my contributions.

### CODE REVIEW

File: playbooks/meta/exporter.yml [hunk 1]
```diff
@@ -9,3 +9,11 @@
 - import_playbook: ../exporter_node_start.yml
 # exporter_start
 # exporter_init
+
```
[the remark this answers is not in the document]
Comment (kpgtek) [reply playbooks/meta/exporter.yml:12]: No playbook files are coded manually.

[the remark this answers is not in the document]
Comment (rpignolet) [reply playbooks/meta/exporter.yml:12]: You should run `tdp playbooks --for-collection tdp_observability` to generate meta playbooks for this collection.

File: tdp_vars_defaults/exporter/exporter.yml [hunk 2]
```diff
@@ -25,3 +25,33 @@ node_exporter_web_listen_address: --web.listen-address=:{{ exporter_node_http_po
...
+
+#####################
+# postgres-exporter #
+#####################
+# postgres-exporter users and group
+postgres_exporter_user: postgres-exporter
+postgres_exporter_group: postgres-exporter
+
+# postgres exporter version
+postgres_exporter_release: postgres_exporter-0.12.0.linux-amd64
```
Comment (gonzaloetjo) [inline tdp_vars_defaults/exporter/exporter.yml:37]: This should be added on tdp-getting-started [tdp-release-uri](https://github.com/TOSIT-IO/tdp-getting-started/blob/master/scripts/tdp-release-uris.txt)

Comment (gonzaloetjo) [reply tdp_vars_defaults/exporter/exporter.yml:37]: Added the PR https://github.com/TOSIT-IO/tdp-getting-started/pull/262

**Comment by kpgtek** [comment]
I forgot to add ```exporter_postgres``` component target for Prometheus.

**Review by rpignolet — APPROVED** [review]

**Review by gonzaloetjo — APPROVED** [review]

**Pull request merged**